scripts/config_check/README.md

```diff
@@ -0,0 +1,118 @@
+regarding each of the options enabled or disabled.
+
+The multi-line `./configure` command with all parameters on subsequent lines is stripped of comments and everything placed on a single line statement when executed.
+
+Upon execution, the entire output is set to a file called [output.txt](./output.txt). Additionally, the enabled/disabled features (those items with an asterisk in the output and the word "yes" or "no") 
+are separated and stored in the respective [Enabled-Features.txt](./Enabled-Features.txt) and [Disabled-Features.txt](./Disabled-Features.txt) files.
+
+## Purpose
+
+This script may help in a variety of ways:
+
+- Observe which options have side-effects that may enable other options.
+- Determine which `--option-setting` causes a specific `#define` to be enabled (helpful for embedded developers).
+- Record specific project settings of enabled and disabled features.
+
+## Installation
+
+There's no need to install this script, as it can be run from the wolfSSL `/scripts/config_check` directory as noted in the Usage section, below.
+
+If you'd like to run from a someplace else, place the  [refresh.sh](./refresh.sh) and [cmd.txt](./cmd.txt) files in a directory and
+edit the respective `refresh.sh` file. The first `cd "../.."` command will need to be edited to change to your wolfssl directory 
+to run the `./configure` command. See the Configuration section, below.
+
+It may be convenient for the command and output files to be in a included in GitHub fork repo for 
+easily tracking changes to the output files, typically your project that is using wolfSSL.
+
+## Configuration
+
+See the variables in the [refresh.sh](./refresh.sh) script:
+
+Note in particular the environment variables:
+
+```
+WOLFSSL_REPO="$PWD"
+WOLFSSL_FILE_ROOT="$WOLFSSL_REPO/scripts/config_check"
+```
+If you'd like to save the results someplace else, the file output locations are set like this:
+
+```
+WOLFSSL_OUTPUT="$WOLFSSL_FILE_ROOT/output.txt"
+WOLFSSL_OPTIONS="$WOLFSSL_FILE_ROOT/options.h"
+WOLFSSL_YES="$WOLFSSL_FILE_ROOT/Enabled-Features.txt"
+WOLFSSL_NO="$WOLFSSL_FILE_ROOT/Disabled-Features.txt"
+```
+
+Edit those locations to suit your needs. See below for more details:
+
+#### `WOLFSSL_REPO` 
+
+This the location of wolfSSL where the `./configure` script should run and is typically the location of your wolfSSL `git clone`. 
+For example in WSL for a clone in the `C:\workspace` directory, this value would be:
+
+`WOLFSSL_REPO="/mnt/c/workspace/wolfssl"`
+
+#### `WOLFSSL_FILE_ROOT`
+
+The directory where `./configure` console output will be saved in various files (a github repo is helpful for tracking changes). 
+For example, you could choose to redirect to a different location
+
+`WOLFSSL_FILE_ROOT="~/myproject/_debug"`
+
+
+#### `WOLFSSL_CMD_FILE`
+
+This the `.configure` command to edit. This is typically the location of the [cmd.txt](./cmd.txt) file

```

Comment:
should be `This is the` not `This the`

Reply:
ah yes, I thought I fixed this, but there was another. good catch. thank you!